tests/ffis.test.ts

```diff
@@ -1,122 +1,148 @@
+      szAuxInfo: "",
+      szMessage: "",
+      szLog: "",
+    });
+    // Methane mol file | Ref: http://www.cheminfo.org/Chemistry/Generate_molfiles/index.html
+    expect(
+      INCHIAPI.MakeINCHIFromMolfileText(
+        `C1CCCC1
+        JME 2016-02-24 Sat Mar 05 09:08:54 GMT+100 2016
+        
+          5  5  0  0  0  0  0  0  0  0999 V2000
+            1.8326    0.0000    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+            2.2652    1.3315    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+            1.1326    2.1544    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+            0.0000    1.3315    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+            0.4326    0.0000    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+          1  2  1  0  0  0  0
+          2  3  1  0  0  0  0
+          3  4  1  0  0  0  0
+          4  5  1  0  0  0  0
+          5  1  1  0  0  0  0
+        M  END
+        `,
+        "",
+        output2.ref()
+      )
+    ).toBe(2);
+    expect(output2.szInChI).toBe("1S/C5H10/c1-2-4-5-3-1/h1-5H2");
 
-const output3 = refNAPI.alloc(inchi_Output, {
-  szInChI: "",
-  szAuxInfo: "",
-  szMessage: "",
-  szLog: "",
-});
-// Ethanol mol file | Ref: http://www.cheminfo.org/Chemistry/Generate_molfiles/index.html
-strict.equal(
-  INCHIAPI.MakeINCHIFromMolfileText(
-    `
-Actelion Java MolfileCreator 1.0
-
-  3  2  0  0  0  0  0  0  0  0999 V2000
-    1.7321   -0.5000   -0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
-    0.8660   -0.0000   -0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
-    0.0000   -0.5000   -0.0000 O   0  0  0  0  0  0  0  0  0  0  0  0
-  2  1  1  0  0  0  0
-  3  2  1  0  0  0  0
-M  END
-`,
-    "",
-    output3
-  ),
-  0
-);
-strict.equal(output3.deref().szInChI, "InChI=1S/C2H6O/c1-2-3/h3H,2H2,1H3");
+    const output3 = refNAPI.alloc(inchi_Output, {
+      szInChI: "1S/C5H10/c1-2-4-5-3-1/h1-5H2",
+      szAuxInfo: "",
+      szMessage: "",
+      szLog: "",
+    });
+    // Ethanol mol file | Ref: http://www.cheminfo.org/Chemistry/Generate_molfiles/index.html
+    expect(
+      INCHIAPI.MakeINCHIFromMolfileText(
+        `C1CCCC1
+        JME 2016-02-24 Sat Mar 05 09:08:54 GMT+100 2016
+        
+          5  5  0  0  0  0  0  0  0  0999 V2000
+            1.8326    0.0000    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+            2.2652    1.3315    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+            1.1326    2.1544    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+            0.0000    1.3315    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+            0.4326    0.0000    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+          1  2  1  0  0  0  0
+          2  3  1  0  0  0  0
+          3  4  1  0  0  0  0
+          4  5  1  0  0  0  0
+          5  1  1  0  0  0  0
+        M  END
+        `,
+        "",
+        output3
+      )
+    ).toBe(2);
+    expect(output3.deref().szInChI).toBe("1S/C5H10/c1-2-4-5-3-1/h1-5H2");
```

Comment:
```suggestion
    expect(output3.deref().szInChI).toBe("InChI=1S/C5H10/c1-2-4-5-3-1/h1-5H2");
```

Should `InChI=` be used? In some cases it is being used. Try to figure why is this inconsistency?